src/app/core/data/data.service.ts

```diff
@@ -90,4 +101,15 @@ export abstract class DataService<TNormalized extends CacheableObject, TDomain>
     // return this.rdbService.buildSingle(href));
   }
 
+  findEmbedded<TEmbeddedNormalized extends CacheableObject, TEmbeddedDomain>(
```

Comment:
In this case we should change browse endpoint for collections now is `/discover/browses/dateissued/collections`

So I will change the code in the way that you have suggested and wait for the rest API implementation. 

We should report this to @abollini and the rest API team